libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -398,10 +388,10 @@ - (void)continueNativeBrowserFlow {
     [self.delegate oauthCoordinator:self didBeginAuthenticationWithSession:_asWebAuthenticationSession];
 }
 
-- (void)beginUserAgentFlow {
+- (void)beginWebViewFlow {
```

Comment:
I was trying to figure out a name to encompass  user-agent and web server, went with this to pair up with `beginNativeBrowserFlow`, open to anything else

Reply:
👍